utils/createElement.js

```diff
@@ -13,10 +13,10 @@ const SlackAuthor = require("../components/SlackAuthor");
 let rootNodeInstance = null;
 
 exports.getHostContextNode = function getHostContextNode(rootNode) {
+  rootNodeInstance = new SlackRoot();
```

Comment:
everything looks for good for me except this line. It will *always* create a `new SlackRoot` even when it's not needed — could you roll this one back?

Reply:
Yeah, sure. I don't thought this problem. Haha!

Reply:
:)